xmodule/static_content.py

```diff
@@ -123,44 +123,33 @@ def _ensure_dir(directory):
 def _write_styles(selector, output_root, classes, css_attribute, suffix):
     """
     Write the css fragments from all XModules in `classes`
-    into `output_root` as individual files, hashed by the contents to remove
-    duplicates
+    into `output_root` as individual files
     """
     contents = {}
 
-    css_fragments = defaultdict(set)
     for class_ in classes:
         class_css = getattr(class_, css_attribute)()
-        for filetype in ('sass', 'scss', 'css'):
-            for idx, fragment_path in enumerate(class_css.get(filetype, [])):
-                with open(fragment_path, 'rb') as fragment_file:
-                    fragment = fragment_file.read()
-                css_fragments[idx, filetype, fragment].add(class_.__name__)
-    css_imports = defaultdict(set)
-    for (idx, filetype, fragment), classes in sorted(css_fragments.items()):  # lint-amnesty, pylint: disable=redefined-argument-from-local
-        fragment_name = "{idx:0=3d}-{hash}.{type}".format(
-            idx=idx,
-            hash=hashlib.md5(fragment).hexdigest(),
-            type=filetype)
-        # Prepend _ so that sass just includes the files into a single file
-        filename = '_' + fragment_name
-        contents[filename] = fragment
+        fragment_names = []
+        for fragment_path in class_css.get('scss', []):
+            with open(fragment_path, 'rb') as fragment_file:
+                fragment = fragment_file.read()
+            fragment_name = "{hash}.{type}".format(
+                hash=hashlib.md5(fragment).hexdigest(),
+                type='scss')
+            # Prepend _ so that sass just includes the files into a single file
+            filename = '_' + fragment_name
```

Comment:
This suggestion doesn't make a big difference in performance or readability but I always suggest to use f-strings, anyways this is part of the original code

Reply:
@andrey-canon I agree that f-strings are better. In this particular case I will leave it as-is because [PR 4/6 will delete this block entirely](https://github.com/openedx/edx-platform/pull/32289/files#diff-773187b16792ffbb7cde812c059dc192bb0e47a0fbe9843121544b0ceceda764) and changing it now would make the rebase more difficult.